### ISSUE TITLE
is this project still developing ?

### ISSUE THREAD

**Issue by eyeix** [thomas4019/pgmongo#4]
the lastest commit had been long gone,
would this project still developing?

**Comment by thomas4019** [comment]
This project is not still developing. I created it as a proof of concept and haven't had the need or time to add more to it. Are you wanting to use it for something?

**Comment by eyeix** [comment]
yeah, use pg as mongodb alternative looks amzing actually, mongodb maintance is more complex than pg,
wish this project can be developed well in the future, 
or is there any similar project in production ?

**Comment by hellower** [comment]
https://github.com/FerretDB/FerretDB

frontend: mongodb wire protocol
backend: postgresql

not yet proudction-ready

**Comment by wwey-payton** [comment]
There is a product on Azure, Azure Cosmos DB for MongoDB (vCore), which provides pg as alternative to mongo.
![image](https://github.com/thomas4019/pgmongo/assets/111406904/a1433950-093a-4fc5-84b3-72e877e4afa5)
